reana_client/validation/utils.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
```

Comment:
Hm, it's a bit problematic because we trigger the validation in `load_reana_spec()` which lives also in `utils.py`. I can't import `utils.py` in validation modules, since it's a circular import

Reply:
moved the `run_command` imports into the functions that uses it